daprdocs/content/en/reference/api/metadata_api.md

```diff
@@ -17,12 +19,21 @@ Component type | Capabilities
 State Store    | ETAG, TRANSACTION, ACTOR, QUERY_API
 Binding        | INPUT_BINDING, OUTPUT_BINDING
 
-## HTTPEndpoints
+### HTTPEndpoints
 Each loaded `HttpEndpoint` provides a name to easily identify the Dapr resource associated with the runtime.
 
-## Attributes
+### Subscriptions
+The metadata API returns a list of pubsub subscriptions that the app has registered with the Dapr runtime. This includes the pubsub name, topic, routes, deadletter topic, and the metadata associated with the subscription.
+
+### Enabled features
+A list of features enabled during build time, this does not include the features from Configuration spec.
```

Comment:
FYI @ItalyPaleAle

Reply:
It’s actually the features from the config spec!

Reply:
I just realized I may have picked the wrong variable. The intent was to have those from configurarion (which does include the build time flags)

Reply:
No problem - I have converted https://github.com/dapr/dapr/pull/6537 to draft for now, I will update the docs once your changes are in.

Reply:
Done

Reply:
The original PR and docs have been updated.